saleor/payment/gateways/stripe/valcome_webhook/views.py

```diff
@@ -19,8 +21,24 @@
 @csrf_exempt
 @transaction_with_commit_on_errors()
 def stripe_webhook(request: ASGIRequest):
+    # Log 2
+    try:
+        sentry_sdk.capture_message("Stripe webhook initiated",
+                                   level="info")
+    except Exception as sentry_error:
+        print(f"Sentry logging failed: {sentry_error}")
+
     body = json.loads(request.body)
 
+    # Log 2
+    try:
+        event_id = body.get('id')
+        print(f"Stripe webhook received - Event ID: {event_id}")
```

Comment:
Soll ma des immer in die Konsole loggen, oder a nur bei am Fehler?

Reply:
jop